drivers/led_strip/Kconfig.ws2812

```diff
@@ -30,11 +30,11 @@ config WS2812_STRIP_I2S
 
 config WS2812_STRIP_GPIO
 	bool "WS2812 LED strip GPIO driver"
-	# Only an Cortex-M0 inline assembly implementation for the nRF51
-	# is supported currently.
+	# Only an Cortex-M0/M4 inline assembly implementation for the nRF51 and
+	# nRF52 is supported currently.
```

Comment:
Although the driver is more generic there still is a hardware dependency towards the GPIO set/clear assembly instructions.